dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/domain/SpanSerializer.kt

```diff
@@ -42,6 +42,7 @@ internal class SpanSerializer(
             metaObject.addProperty(it.key, it.value)
         }
 
+        metaObject.addProperty(TAG_DD_SOURCE, DD_SOURCE_MOBILE)
```

Comment:
Didn't you decide to add this at the Payload (trace) level ?

Reply:
it doesn't seem to work at the payload trace level from what i've seen.